### PR TITLE
there's already a link to MDN in this sentance, remove defunct MDC link

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: apps/browse/templates/browse/search_tools.html [hunk 1]
```diff
@@ -92,7 +92,7 @@
       {# L10n: {0} is an app name, like Firefox. #}
       <li>{{ _('Learn more about the <a href="http://support.mozilla.com/kb/Search+bar">search bar</a> in {0}')|f(APP.pretty) }}</li>
       <li>{{ _('Browse through even more search providers at <a href="http://mycroft.mozdev.org/">mycroft.mozdev.org</a>') }}</li>
-      <li>{{ _('<a href="https://developer.mozilla.org/en/Creating_OpenSearch_plugins_for_Firefox">Make your own</a> at the <a href="https://developer.mozilla.org/">Mozilla Developer Center</a>') }}</li>
+      <li>{{ _('<a href="https://developer.mozilla.org/en/Creating_OpenSearch_plugins_for_Firefox">Make your own</a> search tool') }}</li>
```
Comment (wagnerand) [inline apps/browse/templates/browse/search_tools.html]: We usually call them *search plugins*.

**Comment by wagnerand** [comment]
r+wc

File: apps/browse/templates/browse/search_tools.html (hunk 1 above)
Comment (andymckay) [inline apps/browse/templates/browse/search_tools.html]: The page at https://addons.mozilla.org/en-uS/firefox/search-tools/, which is where this links is - just says "search tools" and not "search plugin".

**Comment by wagnerand** [comment]
MDN calls them "OpenSearch plugins".

**Pull request merged**